server/src/main/java/org/opensearch/cluster/ClusterServiceRequest.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.cluster;
+
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+import org.opensearch.transport.TransportRequest;
+
+import java.io.IOException;
+
+public class ClusterServiceRequest extends TransportRequest {
+    private enum ClusterStateEnum {
```

Comment:
`missingJavadocs` should pass w/o private level javadocs but it's good practice to add them anyway.